src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5483,4 +5483,47 @@ This option should be used when specifying the certificate via -CertificateSubje
     <value>When creating a symbols package, allows to choose between the 'snupkg' and 'symbols.nupkg' format.</value>
     <comment>Do not localize snupkg and symbols.nupkg</comment>
   </data>
+  <data name="TrustedSignersCommandDescription" xml:space="preserve">
+    <value>Provides the ability to manage list of trusted signers located in NuGet.Config</value>
+  </data>
+  <data name="TrustedSignersCommandUsageExamples" xml:space="preserve">
+    <value>nuget trusted-signers
+
+nuget trusted-signers Add -Name existingSource
+
+nuget trusted-signers Add -Name trustedRepo -ServiceIndex https://trustedRepo.test/v3ServiceIndex
+
+nuget trusted-signers Add -Name author1 -CertificateFingerprint CE40881FF5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E039 -FingerprintAlgorithm SHA256
+
+nuget trusted-signers Add -Repository .\..\MyRepositorySignedPackage.nupkg -Name TrustedRepo
+
+nuget-trusted-signers Remove -Name TrustedRepo</value>
+  </data>
+  <data name="TrustedSignersCommandUsageSummary" xml:space="preserve">
+    <value>&lt;List|Add|Remove|Sync&gt; [options]</value>
+  </data>
+  <data name="TrustedSignersCommandAllowUntrustedRootDescription" xml:space="preserve">
+    <value>If specified, chaining to an untrusted root should be allowed for the trusted signer.</value>
+  </data>
+  <data name="TrustedSignersCommandAuthorDescription" xml:space="preserve">
+    <value>If specified, the author signature of the package should be trusted.</value>
```

Comment:
Same feedback as the rest. 
We can discuss/ get feedback from other folks like @dtivel/ @zivkan